tests/windows_test.go

```diff
@@ -69,9 +69,10 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 		util.PanicOnError(err)
 		checks.SkipIfMissingRequiredImage(virtClient, tests.DiskWindows)
 		libstorage.CreatePVC(OSWindows, "30Gi", libstorage.Config.StorageClassWindows, true)
-		windowsVMI = tests.NewRandomVMI()
+		windowsVMI = libvmi.New()
```

Comment:
0bbeba6c9b43a059550724a718927be1fd4ff539 is talking about setting less memory but it only replace `tests.NewRandomVMI()` with `libvmi.New()`. This is not equivalent and the commit-msg doesn't match.

Reply:
Thanks for reviewing.
Done.